messages/config.unset.md

```diff
@@ -27,3 +27,7 @@ You must provide one or more configuration variables to unset. Run "sf config li
 # didYouMean
 
 Did you mean %s?
+
+# unsetGlobalWarning
+
+The %s config is still set globally, unset it by using the --global flag.
```

Comment:
```suggestion
The %s config variable is still set globally, unset it by using the --global flag.
```

Reply:
We use "config variable" in the docs: https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_dev_cli_config_values.htm